code/src/index.css

```diff
@@ -1,13 +1,176 @@
+@import url("https://fonts.googleapis.com/css2?family=Raleway:wght@100;200;300;400;500;600;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  width: 100%;
+  margin: 0 auto;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  box-sizing: border-box;
+  background: #191414;
+  color: #fff;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-family: "Raleway", sans-serif;
+  color: #ebebeb;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/* Designing the whole page here */
+.page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/* Designing the thin grey line top of page */
+.title-container {
+  border-bottom: 1px solid #a0a0a0;
+  color: white;
+  margin-bottom: 20px;
+  padding: 20px 10px;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+/* Designing each card */
+.card {
+  width: 300px;
+  height: 400px;
+  margin: 10px;
+}
+
+.linked-album {
+  position: relative;
+}
+
+.album-name {
+  font-size: 16px;
+  margin-top: 10px;
+}
+
+.linked-artist {
+  text-decoration: none;
+  color: #a0a0a0;
+  font-size: 16px;
+  font-weight: 500;
+  padding-top: 7px;
+}
+
+.icons-container {
+  position: absolute;
+  top: 50%;
+  height: 100%;
+  width: 100%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.icons {
+  display: none;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.icon {
+  width: 40px;
+  padding: 0 20px;
+  filter: brightness(0) invert(1);
+  z-index: 1;
+}
+
+.symbol-color {
+  color: #a0a0a0;
+}
+
+/*---------- HOVER TEXT ------------*/
+.linked-artist:hover {
+  transition: 0.5s ease;
+  color: #fff;
+  text-decoration: underline;
+}
+
+.album-name:hover {
+  transition: 0.5s ease;
+  color: #fff;
+  text-decoration: underline;
+}
```

Comment:
Very small comment, but maybe you could have merged these into one to "save up" code (as they have the same specifics). For instance:

.linked-artist, .album-name: hover{
transition: 0.5s ease;
  color: #fff;
  text-decoration: underline;
}